frontend/src/app/main/data/workspace/libraries.cljs

```diff
@@ -27,10 +27,13 @@
    [app.util.i18n :refer [tr]]
    [app.util.router :as rt]
    [app.util.time :as dt]
+   [app.util.logging :as log]
    [beicon.core :as rx]
    [cljs.spec.alpha :as s]
    [potok.core :as ptk]))
 
+(log/set-level! :warn)
```

Comment:
Why is there a top-level statement here?

Reply:
It's explained in the [logging framework manual](https://github.com/penpot/penpot/blob/develop/docs/02-Frontend-Developer-Guide.md#logging-framework). This call sets the logging level for the current namespace (in this case app.main.data.workspace.libraries).